adobe_vipm/flows/helpers.py

```diff
@@ -705,3 +731,47 @@ def __call__(self, mpt_client, context, next_step):
             return
 
         next_step(mpt_client, context)
+
+
+class ValidateSkuAvailability(Step):
+    """Validate the SKU availability."""
+
+    def __init__(self, *, is_validation: bool) -> None:
+        self.is_validation = is_validation
+
+    def __call__(self, mpt_client, context, next_step):
+        """Validate the SKU availability."""
+        commitment = get_3yc_commitment(context.adobe_customer)
+        if commitment:
```

Comment:
Please put a comment here why you have here +1 year comparison. Is what Stu commented in the ticket.